adapter/pkg/messaging/connection.go

```diff
@@ -212,3 +231,98 @@ type amqpFailoverURL struct {
 	retryCount      int
 	connectionDelay int
 }
+
+func handleEvent(c *Consumer, key string) error {
+	var err error
+
+	logger.LoggerMsg.Debugf("got Connection, getting Channel for %s events", key)
```

Comment:
Can we make this a proper message. Here we do not get the the connection right, kind of misleading message it seems